### PR TITLE
Register warden homer

### PR DESCRIPTION
**Pull request by code423n4**
Auto-generated PR to register the new warden homer

### CODE REVIEW

File: _data/handles/homer.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "handle": "homer",
```
Comment (howiewang17) [inline _data/handles/homer.json:2]: test

**Comment by SamusElderg** [comment]
Can not find a matching user in Discord, will leave the application open for now though

**Comment by sksb2180** [comment]
Warden didn't checked out the channel `i-want-to-be-a-warden` yet.

**Comment by SamusElderg** [comment]
Can not find a matching user in Discord, closing this PR for now

**Pull request closed without merging**